OpenRA.Game/Network/UnitOrders.cs

```diff
@@ -70,6 +70,15 @@ internal static void ProcessOrder(OrderManager orderManager, World world, int cl
 							TextNotificationsManager.AddPlayerJoinedLine(localizedMessage.Key, localizedMessage.Arguments);
 						else if (localizedMessage.Key == Left)
 							TextNotificationsManager.AddPlayerLeftLine(localizedMessage.Key, localizedMessage.Arguments);
+						else if (localizedMessage.Key.Contains("briefing"))
+						{
+							var map = orderManager.LobbyInfo.GlobalSettings.Map;
+							if (map == null && localizedMessage.Arguments.ContainsKey("map"))
+								map = (string)localizedMessage.Arguments.GetValueOrDefault("map");
```

Comment:
this doesn't make sense. Null map can still go through and crash.

Reply:
It makes sense if you look at the situations where map is null and when it is triggered, but sure, I can write this more defensively.